### ISSUE TITLE
WASM Input (needs to be better)

### ISSUE THREAD

**Issue by simbleau** [mvlabat/bevy_egui#246]
This crate has a historic battle to handling input correctly on WASM. These problems make up a sizeable majority of unresolved issues:
- #196 
- #169 
- #199 
- #185 
- #145 
- #113 
- #101 
- #29 

As I understand it, bevy recently updated to winit 0.29 - which brought consistent APIs to physical keys vs. logical keys, right? (p.s. thanks @Vrixyz - https://github.com/bevyengine/bevy/pull/10702)

Egui just released 0.25, which dually brought in changes to reflect the new winit 0.29 update.

I think we're now unblocked.

These are the issues I face daily working in a completely WASM codebase using bevy_egui and would love to see prioritized
- 
- #113 (copy/paste)
- #196 (support for non-qwerty keyboards)
- Emoji picker doesn't open in the correct place on text inputs. It always opens in the middle of the screen space.
  - Desired behavior, like in egui.rs:
  <img width="386" alt="image" src="https://github.com/mvlabat/bevy_egui/assets/48108917/6269cc32-4610-4d0e-b5ec-6336bf17fd38">

**Comment by v-kat** [comment]
Does https://github.com/mvlabat/bevy_egui/pull/236 help with any of these issues? I was hoping the winit upgrade on bevy main would

**Comment by Vrixyz** [comment]
> Does #236 help with any of these issues? I was hoping the winit upgrade on bevy main would

Yes, alongside https://github.com/bevyengine/bevy/pull/11400 ; I think we might be able to drive https://github.com/mvlabat/bevy_egui/pull/178 to completion 🤞

**Comment by bananaturtlesandwich** [comment]
i've got a website up at https://skyweb.spuds.casa which this is pretty critical for

**Comment by vladbat00** [comment]
Also, maybe users can setup more fine-grained event handling on their own, the documentation in this winit PR might help: https://github.com/rust-windowing/winit/pull/3451
